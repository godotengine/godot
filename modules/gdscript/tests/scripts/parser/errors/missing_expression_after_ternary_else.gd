func test():
	var x = 1 if false else
	print("oops")
	print(x)
