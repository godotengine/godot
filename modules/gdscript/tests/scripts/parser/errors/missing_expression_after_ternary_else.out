GDTEST_PARSER_ERROR
Expected expression after "else".
